src/components/Chat/InputMessageV2.tsx

```diff
@@ -194,24 +401,44 @@ const InputMessageV2 = ({
             labelLeft={!inputFocus ? 'Anonymity' : undefined}
             styleLabelLeft={[
               styles.labelToggle,
-              {color: type === 'ANONYMOUS' ? colors.gray : colors.blue}
+              {color: type === ANONYMOUS ? colors.gray : colors.blue}
             ]}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `toggleChange` function has been renamed to `toggleChange` and now uses constants for the chat types instead of hardcoded strings. This is a good change as it reduces the risk of typos and makes the code easier to maintain.



<!-- This is an auto-generated comment by OSS CodeRabbit -->